### ISSUE TITLE
If a user has a discord name in their profile info have the bot DM them when there are new posts?

### ISSUE THREAD

**Issue by michaelplzno** [mesosoi/silverwaregames-io-issue-tracker#238]
Is that possible?

**Comment by michaelplzno** [comment]
your solution is perfect, faster is better too!

**Comment by mesosoi** [comment]
I formerly thought that you meant that if a user was mentioned in a post, then they should be mentioned in Discord, too (if they have specified their Discord profile in the Settings), but now I'm not sure. Do you want to mention all SWGIO users in Discord once there's any new post? This can be achieved even easier!

**Comment by michaelplzno** [comment]
No, when someone (at)'s a user add their discord name ping in the message.

**Comment by michaelplzno** [comment]
Also, if someone has their discord name in their profile, ping them in discord when someone likes their post, and ping them when someone replies to their post.
